cms/src/components/content/EditContentParentValidation.vue

```diff
@@ -139,6 +157,7 @@ watch(
                         :languages="props.languages"
```

Comment:
Add a sort function to languages prop like something like this?
`:languages="languages.sort(sortByName)"`

It is also not necessary to write "props.xxx" in the template section.